namespaces/live-1.cloud-platform.service.justice.gov.uk/check-my-diary-preprod/resources/ecr.tf

```diff
@@ -0,0 +1,39 @@
+module "checkmydiary-service-preprod" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.4"
+  repo_name = "check-my-diary-preprod"
```

Comment:
This will create an image in `<ecr-endpoint>/check-my-diary/check-my-diary-preprod` - are you planning on using different ECR repos per environment?

Reply:
yes - that's the exact intention - we need / want them all to be independant of any name space